modules/client/src/tokenVoting/internal/utils.ts

```diff
@@ -335,3 +336,27 @@ export function computeProposalStatusFilter(status: ProposalStatus) {
   }
   return where;
 }
+
+export async function isERC20Token(
```

Comment:
Let's add a comment that this is a insufficient check. 
Feel free to add more checks to make this more satisfiable, as you did with the tokenURI in this or an upcoming task (WDYT @Rekard0?)

As this happens once during creation, I am fine with doing multiple static calls (in a Promise.all) and this being slow overall.